pkg/deploy/util/util.go

```diff
@@ -375,13 +504,17 @@ func MakeDeploymentV1(config *deployapi.DeploymentConfig, codec runtime.Codec) (
 	for k, v := range config.Spec.Template.Labels {
 		podLabels[k] = v
 	}
+	// NOTE: You need to update the labelsToSkip variable in pkg/deploy/util when
+	// adding a new label here.
```

Comment:
@kargakis guess this check is enough? i don't think we want to keep adding more annotations or labels but if we do, this will remind you to update the list.